Modules/FIT/FT0/src/DigitQcTask.cxx

```diff
@@ -628,9 +628,9 @@ void DigitQcTask::monitorData(o2::framework::ProcessingContext& ctx)
     switch (mTrgModeSide) {
       case TrgModeSide::kAplusC:
         if (mTrgModeThresholdVar == TrgModeThresholdVar::kAmpl) {
-          if (pmSumAmplA + pmSumAmplC >= 2 * mTrgThresholdCenA)
+          if (pmSumAmplA + pmSumAmplC > 2 * mTrgThresholdCenA)
             mMapTrgSoftware[o2::ft0::Triggers::bitCen] = true;
-          else if (pmSumAmplA + pmSumAmplC >= 2 * mTrgThresholdSCenA)
+          else if (pmSumAmplA + pmSumAmplC > 2 * mTrgThresholdSCenA && !mMapTrgSoftware[o2::ft0::Triggers::bitCen])
```

Comment:
@arvindkhuntia why did you add `... && !mMapTrgSoftware[o2::ft0::Triggers::bitCen]` ? there are already `if-else` construction, where first `if` checks `bitCen`

Reply:
Hi @afurs, you are right, just removed it.